Complex/ComplexStruct.UnitTest/ComplexTests.cs

```diff
@@ -0,0 +1,68 @@
+using NUnit.Framework;
+using System;
+using Complex;
+
+
+namespace Complex
+{
+    [TestFixture]
+    public class ComplexTests
+    {
+        [Test]
+        public void ConstructorTest()
+        {
+            var complex = new Complex(2, 5);
+
+            Assert.That(complex.Re, Is.EqualTo(2));
+            Assert.That(complex.Im, Is.EqualTo(5));
+            Assert.That(complex.Abs, Is.EqualTo(Math.Sqrt(29)));
+        }
+
+        [Test]
+        public void ToStringTest()
+        {
+            var complex1 = new Complex(2, 1);
+            var complex2 = new Complex(2, -1);
+            var complex3 = new Complex(0, 0);
+            var complex4 = new Complex(0, 5);
+            var complex5 = new Complex(2, 0);
+            var complex6 = new Complex(2, -5);
+            var complex7 = new Complex(2, 5);
+
+
+            Assert.That(complex1.ToString(), Is.EqualTo("2 + i"));
+            Assert.That(complex2.ToString(), Is.EqualTo("2 - i"));
+            Assert.That(complex3.ToString(), Is.EqualTo("0"));
+            Assert.That(complex4.ToString(), Is.EqualTo("5i"));
+            Assert.That(complex5.ToString(), Is.EqualTo("2"));
+            Assert.That(complex6.ToString(), Is.EqualTo("2 - 5i"));
+            Assert.That(complex7.ToString(), Is.EqualTo("2 + 5i"));
+        }
```

Comment:
Этот тест лучше было сделать параметризованным
```suggestion
        [TestCase(2, 1, "2 + i")]
        [TestCase(2, -1, "2 - i")]
        [TestCase(0, 0, "0")]
        [TestCase(0, 5, "5i")]
        [TestCase(2, 0, "2")]
        [TestCase(2, -5, "2 - 5i")]
        [TestCase(2, 5, "2 + 5i")]
        public void ToStringTest(doiuble a, double b, string expected)
        {
            var z = new Complex(a, b);

            Assert.That(z.ToString(), Is.EqualTo(expected));      
        }
```